oak/server/enclave_server.h

```diff
@@ -168,11 +86,11 @@ class EnclaveServer final : public asylo::TrustedApplication {
   // The port on which the server is listening.
   int port_;
 
-  std::unique_ptr<::oak::OakNode> node_;
-  std::shared_ptr<::grpc::ServerCredentials> credentials_;
+  std::unique_ptr<OakNode> node_;
+  std::shared_ptr<grpc::ServerCredentials> credentials_;
 
   ::grpc::AsyncGenericService module_service_;
-  std::unique_ptr<::grpc::ServerCompletionQueue> completion_queue_;
+  std::unique_ptr<grpc::ServerCompletionQueue> completion_queue_;
```

Comment:
Please leave the :: prefix for namespaces other than std::.  You can add it in other places for consistency.